pkg/logger/logger.go

```diff
@@ -117,11 +119,21 @@ func zapSugaredLoggerTo(options *Opts) *zap.SugaredLogger {
 
 	if options.Console {
 		encCfg := zap.NewDevelopmentEncoderConfig()
-		encCfg.LevelKey = ""
-		encCfg.TimeKey = ""
-		encCfg.NameKey = ""
-		encCfg.CallerKey = ""
-		enc = zapcore.NewConsoleEncoder(encCfg)
+		if options.OutputFormat == "json" {
+			encCfg.CallerKey = util.GetOSEnv("CALLER_KEY", encCfg.CallerKey)
```

Comment:
Could we set some defaults "comprehensible" keys ?

Like `level`, `message`, `time`... (would avoid uppercase for the defaults)

Reply:
Defaults are now caller, level, message, name, stacktrace, and time (all lowercase). For example:
`{"level":"INFO","time":"2020-02-25T12:12:10.644-0500","name":"kogito-cli","message":"Using project 'testns1'"}`